### PR TITLE
fix(forms-web-app): allow GA_ID to be set dynamically from env var

### PR DESCRIPTION
**Pull request by frank-dodion**
## Ticket Number
AS-1439

## Description of change
Allow GA_ID to be set dynamically from env var

### CODE REVIEW

**Review by MrSimonEmms — APPROVED** [review]

**Review by MrSimonEmms — DISMISSED** [review]
Can you use the config file?

**Comment by Fenisay** [comment]
My understanding of what was supposed to be done is that the good id should be exposed somewhere and called dynamically at runtime by the app.

**Comment by frank-dodion** [comment]
> My understanding of what was supposed to be done is that the good id should be exposed somewhere and called dynamically at runtime by the app.

I have demo'd this locally with @a6software and it looks like the correct GA_ID is available client side.

File: packages/forms-web-app/webpack.dev.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ module.exports = merge(common, {
   devtool: 'inline-source-map',
   plugins: [
     new webpack.DefinePlugin({
-      'process.env.GOOGLE_ANALYTICS_ID': JSON.stringify('test'),
+      'process.env.GOOGLE_ANALYTICS_ID': JSON.stringify(process.env.GOOGLE_ANALYTICS_ID),
```
[the remark this answers is not in the document]
Comment (frank-dodion) [reply packages/forms-web-app/webpack.dev.js]: Done

**Review by MrSimonEmms — APPROVED** [review]

**Pull request merged**